src/main/scala/com/github/shadowsocks/ProfileManagerActivity.scala

```diff
@@ -111,12 +102,9 @@ class ProfileManagerActivity extends AppCompatActivity with OnMenuItemClickListe
       }
     }
 
-    def onClick(v: View) = {
+    def onClick(v: View) {
       ShadowsocksApplication.switchProfile(item.id)
       finish
-      if (isLauncherFromShareIntent){
-        TaskStackBuilder.create(ProfileManagerActivity.this).addNextIntentWithParentStack(NavUtils.getParentActivityIntent(ProfileManagerActivity.this)).startActivities()
```

Comment:
When user launch ProfileManagerAcitivity from share intent, after he/she added profile and clicked to the new profile, just simplely finish the activity may exit the app, user probably want to back to Shadowsocks activity to enable it. I think we should keep this logic or use something else instead (like automatic enable it when launch from share intent).